### DISCUSSION TITLE
Anyone else getting greatly reduced download speeds when scraping?

### DISCUSSION THREAD

**Discussion by purpleemoji** [UltimaHoarder/UltimaScraper#1373] [General]
Has anyone noticed a speeds when scraping are much lower than they were previously? I'm getting a very slow cap at ~1.8MB/s whereas previous download speeds were much faster (I'd seen 80+MB/s previously).  This occurs at relatively low traffic times on OF also.

I'm using gigabit ethernet and based on the speed tests I've done, bandwidth is not an issue. Also I have not made any network or router setting changes that would limit the speed.

Just curious if others are seeing this. Seems like its from the OF side of things.

**Comment by UltimaHoarder** [comment]
It may be because the script is downloading images?

Video files tend to use max speed.

**Reply by soulburn74** [reply]
Its no longer separating Picture downloads from Video Downloads, and downloading both in the same process. (Vids download at max speed, pics are so small, that they don't have time to ramp up to max speed, so you are seeing the avg of the two mixed together)

**Reply by dkwtfigo** [reply]
> Its no longer separating Picture downloads from Video Downloads, and downloading both in the same process. (Vids download at max speed, pics are so small, that they don't have time to ramp up to max speed, so you are seeing the avg of the two mixed together)

But the average of both can't be close to zero right? My speeds are always dropping down to nothing almost instantly and I can't find an explanation for it...very frustrating. Even if I choose to download only pictures or only videos, it's the same.

**Comment by thepspgamer** [comment]
I have also found this. Downloads used to be 15-20mbps at least. Now, they seem to be capped at like 4-5mbps.

Not a huge issue, just something I noticed

**Comment by dkwtfigo** [comment]
Anyone else getting absolutely abysmal download speeds?
I haven't used the script in half a year or so, downloaded the latest version recently because I resubbed some models and I'm wondering if it is OF detecting that you're scraping or whatever and thus capping your speed drastically.
Now I don't have a fast connection to begin with (1.7mbps max lol) but it starts off normally, manages to download a file or two and then will drop to single digit kB/s or even B/s speeds instantly. No matter the time of day it's always the same...are OF servers just complete shite or could it also be a settings issue?

**Reply by LunarPenguin** [reply]
OF is not known for their speed or their stability so that may also contribute to the speeds you're seeing

**Reply by dkwtfigo** [reply]
> OF is not known for their speed or their stability so that may also contribute to the speeds you're seeing

But it’s in the B/s range, that can’t be normal.
I normally use the script on macOS.
But I got so fed up with the speeds I had to try it on a different machine, so set it all up on windows and surprisingly enough it downloads at constant max speed.
Whatever the reason behind that might be…
